packages/core/src/components/MarkdownContent/MarkdownContent.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+import ReactMarkdown from 'react-markdown';
+import gfm from 'remark-gfm';
+import React from 'react';
+import { BackstageTheme } from '@backstage/theme';
+import { CodeSnippet } from '../CodeSnippet';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  markdown: {
+    '& table': {
+      borderCollapse: 'collapse',
+      border: `1px solid ${theme.palette.border}`,
+    },
+    '& th, & td': {
+      border: `1px solid ${theme.palette.border}`,
+      padding: theme.spacing(1),
+    },
+    '& td': {
+      wordBreak: 'break-word',
+      overflow: 'hidden',
+      verticalAlign: 'middle',
+      lineHeight: '1',
+      margin: 0,
+      padding: theme.spacing(3, 2, 3, 2.5),
+      borderBottom: 0,
+    },
+    '& th': {
+      backgroundColor: theme.palette.background.paper,
+    },
+    '& tr': {
+      backgroundColor: theme.palette.background.paper,
+    },
+    '& tr:nth-child(odd)': {
+      backgroundColor: theme.palette.background.default,
+    },
+
+    '& a': {
+      color: theme.palette.link,
+    },
+    '& img': {
+      maxWidth: '100%',
+    },
+  },
+}));
+
+/**
+ * MarkdownContent. Renders markdown (CommonMark, optionally with [GFM](https://github.com/remarkjs/remark-gfm)) to formatted HTML.
+ */
+type Props = {
+  content: string;
+  enableGfm?: boolean;
```

Comment:
Given the possible list of things here, should we make this `dialect="gfm"` instead? Agree that that could be default though, and one can use `dialect="common-mark"` if needed

Reply:
Although, from the looks of it`enableGfm` maps to `plugins` which I'm taking as `remark` plugins which can add individual features rather then the dialect. I suppose `dialect` could be union with a future plugins later on though.

Reply:
Yeah setting `dialect="common-mark"` would translate to no plugins. I don't think we need a 1-to-1 mapping between dialects and remark plugins or anything like that. The dialect prop would just expose the possible dialects, and we may or may not stick to only those two forever.

Reply:
Thanks for the participation and the good ideas. I think it is not excluded that we invent our own dialects. A combination of different plugins could result into a new dialect. Adding the plugin [remark-emoji](https://github.com/rhysd/remark-emoji ) could result into the new dialect `"common-mark-emoji"` and so on...

I'm going to add this in this PR 👍